File: 김형규/2019_01_29/2019_01_29_code.js
function solution(numbers) {
    const answer = numbers.sort((a, b) => parseInt(b+''+a)-parseInt(a+''+b)).join('');
    if(answer[0] === '0')
        return '0';
    return answer;
}

console.log(solution([3, 30, 34, 5, 9]));

File: 안남규/2019-01-29_code.py
def solution(heights):
    answer = []
    for i in range(len(heights)-1, -1, -1):
        for j in range(i-1, -1, -1):
            if heights[i] < heights[j]:
                answer.append(j+1)
                break
        else:
            answer.append(0)

    return answer[::-1]